core/object/script_language_extension.h

```diff
@@ -101,6 +101,8 @@ class ScriptExtension : public Script {
 	EXBIND1RC(bool, has_method, const StringName &)
 	EXBIND1RC(bool, has_static_method, const StringName &)
 
+	EXBIND1RC(int, get_method_argument_count, const StringName &)
```

Comment:
Could remove this part from the extension and rely on `get_method_info` but it might not provide all the necessary details, but it would make extending easier and only affect the `Callable` part

Just as an option to reduce the extent of extension changes